src/user/ci-skeleton.rst

```diff
@@ -55,7 +55,7 @@ not running ``myproj`` CI as a regular feedstock. The ``.gitignore`` is
 modified to not accidentally add unwanted conda-smithy temporary files
 to your repository.
 
-Also the steps that the ``ci-skeleton`` spits out are very important
+Also the steps that the ``ci-skeleton`` splits out are very important
```

Comment:
Dunno about this. 
'spits out' might be correct. 
'the steps that the ``ci-skeleton`` spits out...' would mean the steps this command displays.

Reply:
Yeah I am not very sure either. But let @viniciusdc check it once.

Reply:
It's indeed a good point, as we are referring to these commands both before and after this phrase in different contexts. But as we are directing the user to read more about the command spitted out by `ci-skeleton`, I think it would be better to maintain "spit" for this.